debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/MySqlAntlrDdlParser.java

```diff
@@ -50,6 +50,7 @@ public class MySqlAntlrDdlParser extends AntlrDdlParser<MySqlLexer, MySqlParser>
     private final ConcurrentMap<String, String> charsetNameForDatabase = new ConcurrentHashMap<>();
     private final TableFilter tableFilter;
     private final BinlogCharsetRegistry charsetRegistry;
+    private static final DataTypeResolver dataTypeResolver = initializeDataTypeResolver();
```

Comment:
Are you sure you intended the variable to be static here? I'd prefer we avoid that much like you have done in the MariaDB and Oracle implementations.

Reply:
I think i intended them to be static only. This resolver should be bound to the type of parser right and not the specifc instantiated object ? 

If we don't keep it static, two objects of MySqlAntlrDdlParser.java will both have their own objects of resolver. However, it doesn't add much value and increases memory footprint ?

Reply:
@akanimesh7 I'd prefer to have it as non-static attribut esp. as you mentioned the parsers are instatiated only once.